.github/ISSUE_TEMPLATE/4-flaky-test.yml

```diff
@@ -0,0 +1,45 @@
+name: "❄️ Flaky internal test"
```

Comment:
This is going to be public, right? Intended for our use - but on the Cypress repo, so users will be able to see it and create issues using it.

Perhaps use a title like "Flaky test in `cypress-io/cypress` repository"? "Internal" may not be descriptive enough for people who don't already know what it means.